controlplane/go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/networkservicemesh/networkservicemesh/controlplane
 
 require (
+	github.com/fsnotify/fsnotify v1.4.7
```

Comment:
Is this being pulled through by viper?

Reply:
It's used by NewPrefixHelper() to monitor config file updates